hrapplications/views.py

```diff
@@ -28,11 +28,11 @@ def hr_application_management_view(request):
 		# Get the corp the member is in
 		auth_info = AuthServicesInfo.objects.get(user=request.user)
 		if auth_info.main_char_id != "":
-			main_alliance_id = EveManager.get_charater_alliance_id_by_id(auth_info.main_char_id)
-			if main_alliance_id == settings.ALLIANCE_ID:
+			main_corp_id = EveManager.get_charater_corporation_id_by_id(auth_info.main_char_id)
+			if main_corp_id == settings.CORP_ID:
```

Comment:
logic problem, will only return results to members of executor corp, these results being for entire alliance